lib/api/src/grpc/proto/points_internal_service.proto

```diff
@@ -241,7 +241,7 @@ message QueryShardPoints {
     oneof score {
       RawQuery vector = 1; // (re)score against a vector query
       bool rrf = 2; // Reciprocal Rank Fusion
```

Comment:
Not related to this PR, but it might be better to use 
```
enum Fusion {
  RRF
}
```
instead of boolean?

Reply:
I considered that but I thought it would be too much nesting. Considering I wasn't aware if the other fusion techniques are configurable

Reply:
booleans are much harder to extend